Configurations/unix-Makefile.tmpl

```diff
@@ -324,11 +324,15 @@ LANG=C
 {- dependmagic('build_modules'); -}: build_modules_nodep
 {- dependmagic('build_programs'); -}: build_programs_nodep
 
-build_generated: $(GENERATED_MANDATORY)
+build_generated: $(GENERATED_MANDATORY) build_doc
 build_libs_nodep: libcrypto.pc libssl.pc openssl.pc
 build_modules_nodep: $(MODULES)
 build_programs_nodep: $(PROGRAMS) $(SCRIPTS)
 
+build_doc:
+	$(PERL) "-I$(BLDDIR)" -Mconfigdata "util/dofile.pl" \
+	    "-oMakefile" -s doc/perlvars.pl -i.in doc/man1/*.in
+
```

Comment:
Don't ignore Windows and VMS.

Reply:
I don't know the best way to do them, help needed.